### DISCUSSION TITLE
ccf-validation-tools/Heart/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#268] [General]
# ccf-validation-tools/Heart/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Heart/comments/

**Comment by emquardokus** [comment]
Heart_v1.3_DRAFT
https://docs.google.com/spreadsheets/d/107Mt6KUlHxQIv3nuly6Mv8XdmxBCmgA7xjv1PEcfrqk/edit#gid=1759721736

Edits 9-13-2023 AFTER validation had occurred; Validation report will reflect change next week 9-20-2023 Ellen

**Different labels**

**Fixed  rows 134, 135, 136**:  In row 133, the term UBERON:0006958 has different name/label in the source ontology. The name/label in the ASCT+B table is great cardiac vein and the one in the ontology is **great vein of heart**. For reference, the given name/label by SMEs is great cardiac vein. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Fixed Rows 197-204**    In row 196, the term UBERON:0000946 has different name/label in the source ontology. The name/label in the ASCT+B table is Cardial valve and the one in the ontology is **cardiac valve**. For reference, the given name/label by SMEs is Cardial valve. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Fixed**    In row 142, the term UBERON:0002353 has different name/label in the source ontology. The name/label in the ASCT+B table is atrioventricular bundle and the one in the ontology is **bundle of His**. For reference, the given name/label by SMEs is bundle of His. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Fixed in rows 21, 61, 105, 153, 159, 213**    In row 21, the term CL:0002138 has different name/label in the source ontology. The name/label in the ASCT+B table is lymphatic endothelial cell and the one in the ontology is endothelial cell of lymphatic vessel. For reference, the given name/label by SMEs is lymphatic endothelial cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.


**Blank ontology ID**

**Added valve interstitial cell	CL:4030032**
[NTR] valve interstitial cell (heart) #1736 https://github.com/obophenotype/cell-ontology/issues/1736
In row 196, no term id was found for the name/label valve interstitial cell.
In row 198, no term id was found for the name/label valve interstitial cell.
In row 200, no term id was found for the name/label valve interstitial cell.
In row 202, no term id was found for the name/label valve interstitial cell.

**In row 29, no term id was found for the name/label vasculature.**
To fix issue; original table had vasculature, capillaries, venules with no mapping. 
Change vasculature to capillary capillary	UBERON:0001982
Add row, change vasculature to venule venule	UBERON:0001979


**septal perforating arteries term not in Uberon; refers to several arteries supplying interventricular septum	UBERON:0002094**
**Emailed Griffin Weber to add and map to blood vasculature table see citations below**
In row 130, no term id was found for the name/label septal perforating arteries.
In row 131, no term id was found for the name/label septal perforating arteries.
In row 132, no term id was found for the name/label septal perforating arteries.

The septal perforating arteries are vessels that typically branch off the anterior and inferior interventricular arteries. They arise approximately at right angles, penetrate and supply the interventricular septum.

Ferraro, Daniel, Iolanda Aquila, Lorenzo Volpe, Ilaria Purri, Eugenia Pasceri, Claudia Chiarello, Giuseppina Mascaro, Maria Petullà, Annalisa Mongiardo, and Ciro Indolfi. 2022. “911 THE HYPERTROPHIC SEPTAL PERFORATING BRANCH MISINTERPRETED AS INTERVENTRICULAR DEFECT.” European Heart Journal Supplements 24 (Supplement_K): suac121.183. https://doi.org/10.1093/eurheartjsupp/suac121.183.

Kim, Sung Deuk. 2019. “Anatomy of the Septal Perforating Arteries of the Heart.” Anatomy & Cell Biology 52 (3): 236–41. https://doi.org/10.5115/acb.18.175.

Hosseinpour, A.R., R.H. Anderson, and S.Y. Ho. 2001. “The Anatomy of the Septal Perforating Arteries in Normal and Congenitally Malformed Hearts.” The Journal of Thoracic and Cardiovascular Surgery 121 (6): 1046–52. https://doi.org/10.1067/mtc.2001.113604.

Bream, P. R., A. S. Jr Souza, L. P. Elliott, B. Soto, and G. C. Curry. 1979. “Right Superior Septal Perforator Artery: Its Angiographic Description and Clinical Significance.” AJR. American Journal of Roentgenology 133 (1): 67–73. https://doi.org/10.2214/ajr.133.1.67.

Wasilewski, Jarosław, Jacek Niedziela, Tadeusz Osadnik, Agata Duszańska, Wojciech Sraga, Piotr Desperak, Jolanta Myga-Porosiło, Zuzanna Jackowska, Andrzej Nowakowski, and Jan Głowacki. 2015. “Predominant Location of Coronary Artery Atherosclerosis in the Left Anterior Descending Artery. The Impact of Septal Perforators and the Myocardial Bridging Effect.” Kardiochirurgia i Torakochirurgia Polska/Polish Journal of Thoracic and Cardiovascular Surgery 12 (4): 379–85. https://doi.org/10.5114/kitp.2015.56795.

**Comment by emquardokus** [comment]
Heart v1.4 https://docs.google.com/spreadsheets/d/1IDAwYT1EW1Ao9fuNsZK8_74K24ZHlg7xFVpOjc6cI7E/edit#gid=1759721736

**Different labels**
FIXED---this was a change made in CL original term fat cell, changed to adipocyte in this last quarter, which is why this became an issue. The term CL:0000136 has a different name/label in the source ontology in the following 1 row 37. The name/label in the ASCT+B table is fat cell and the one in the ontology is adipocyte. For reference, the given name/label by SMEs is adipocyte. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Additional changes to table**
1. Removed NTR for valve interstitial cell	CL:4030032 from 3 places in NOTES
2. Update citation format and REF/1/ID columns from REF/1/DOI

**Comment by emquardokus** [comment]
Heart_v1.5_DRAFT
https://docs.google.com/spreadsheets/d/1kDn3zoY_5NmGCfemBLdshDT7JVUWtEMSGBrD4jsHeK0/edit?gid=1759721736#gid=1759721736

**Different labels**

1.  updated 6 instances, recently changed in uberon. The term CL:0000136 has a different name/label in the source ontology in the following 5 rows 37, 48, 90, 103, 186. The name/label in the ASCT+B table is fat cell and the one in the ontology is adipocyte. For reference, the given name/label by SMEs is adipocyte. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

 2. Fixed--removed space from end of 4 instances of mast cell. The term CL:0000097 has a different name/label in the source ontology in the following 4 rows 104, 152, 158, 212. The name/label in the ASCT+B table is mast cell _ and the one in the ontology is _mast cell. For reference, the given name/label by SMEs is mast cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
Heart_v1.5_DRAFT
https://docs.google.com/spreadsheets/d/1kDn3zoY_5NmGCfemBLdshDT7JVUWtEMSGBrD4jsHeK0/edit?gid=1759721736#gid=1759721736

**Blank ontology ID**
1. Fixed Uberon ID added UBERON:8600063 septal perforating artery. No term id was found for the name/label septal perforating arteries in the following 3 rows 131, 132, 133.

**Terms from another ontology**
1. updated 3 instances of left anterior descending artery UBERON:8600102 previously FMA:3862
2.updated 4 instances cardiac muscle of right atrium UBERON:0003379 previously FMA:83531
3.  updated 8 instances cardiac muscle of left atrium UBERON:0003380 previously FMA:83532
4. updated 22 instances cardiac muscle tissue of interventricular septum UBERON:0003383 previously FMA:83539
5. updated 3 instances of conus artery UBERON:0039232 previously FMA:15736
6. updated 4 instance of previously right ventricle myocardium UBERON:0006567 previously FMA:9535
7. updated 10 instances of moderator band UBERON:8470002 previously  FMA:7272
